kustomize/components/memorystore/README.md

```diff
@@ -47,7 +47,7 @@ components:
 ```
 
 Update current Kustomize manifest to target this Memorystore (Redis) instance.
-```bash
+```sh
 REDIS_IP=$(gcloud redis instances describe redis-cart --region=${REGION} --format='get(host)')
 REDIS_PORT=$(gcloud redis instances describe redis-cart --region=${REGION} --format='get(port)')
 sed -i "s/{{REDIS_ADDR}}/${REDIS_IP}:${REDIS_PORT}/g" components/memorystore/kustomization.yaml
```

Comment:
**Suggestion:** Should we update this `sed` command?
```suggestion
sed -i "s/REDIS_ADDR/${REDIS_IP}:${REDIS_PORT}/g" components/memorystore/kustomization.yaml
```
Regarding: https://github.com/GoogleCloudPlatform/microservices-demo/pull/1102#discussion_r1001900094

Reply:
Oh yeah, maybe a merge issue, let me check that and update this.